aap_compose_dev.yaml

```diff
@@ -88,24 +91,45 @@ services:
     volumes:
       - "etc_pulp_certs:/etc/pulp/certs"
       - "var_lib_pulp:/var/lib/pulp"
+      - ".:/src/galaxy_ng"
     command: |
       bash -c "
-        if [[ ! -e /etc/pulp/certs/database_fields.symmetric.key ]] || [[ -s /etc/pulp/certs/database_fields.symmetric.key ]]; then
-          mkdir -p /etc/pulp/certs/;
-          echo 'check openssl and install ...';
-          rpm -q openssl || dnf -y install openssl;
-          echo 'generate key ...';
-          openssl rand -base64 32 > /etc/pulp/certs/database_fields.symmetric.key;
-          echo 'chown key ...';
-          chmod 640 /etc/pulp/certs/database_fields.symmetric.key;
-        else
-          echo 'symmetric key exists'
-        fi;
+        echo '#> STEP: Database Symmetric Key';
+        echo 'WARNING: Symmetric key is hardcoded for development only.';
+        echo 'DNmNdwgyZugTax9S64J0FITTr9IHPxbuoF1F1CGPr68=' > /etc/pulp/certs/database_fields.symmetric.key;
         find /etc/pulp ;
         echo '# KEY >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>';
         cat /etc/pulp/certs/database_fields.symmetric.key;
         echo '<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<';
-        echo "DONE!";
+        echo 'DONE! <#';
+
+        echo '#> STEP: Signing scripts';
+        base64 -d <<< 'IyEvdXNyL2Jpbi9lbnYgYmFzaApHTlVQR0hPTUU9L3Jvb3QvLmdudXBnLwpGSUxFX1BBVEg9JDEKU0lHTkFUVVJFX1BBVEg9IiQxLmFzYyIKQURNSU5fSUQ9IiRQVUxQX1NJR05JTkdfS0VZX0ZJTkdFUlBSSU5UIgpQQVNTV09SRD0iR2FsYXh5MjAyNCIKZ3BnIC0tbG9jay1uZXZlciAtLXF1aWV0IC0tYmF0Y2ggLS1waW5lbnRyeS1tb2RlIGxvb3BiYWNrIC0teWVzIFwKICAtLXBhc3NwaHJhc2UgJFBBU1NXT1JEIC0taG9tZWRpciAvcm9vdC8uZ251cGcvIC0tZGV0YWNoLXNpZ24gLS1hcm1vciBcCiAgLS1vdXRwdXQgJFNJR05BVFVSRV9QQVRIICRGSUxFX1BBVEgKU1RBVFVTPSQ/CmlmIFsgJFNUQVRVUyAtZXEgMCBdOyB0aGVuCiAgIGVjaG8ge1wiZmlsZVwiOiBcIiRGSUxFX1BBVEhcIiwgXCJzaWduYXR1cmVcIjogXCIkU0lHTkFUVVJFX1BBVEhcIn0KZWxzZQogICBleGl0ICRTVEFUVVMKZmk=' > /var/lib/pulp/scripts/collection_sign.sh;
+        base64 -d <<< 'IyEvdXNyL2Jpbi9lbnYgYmFzaAplY2hvICJHYWxheHkyMDI0IiA+IC90bXAva2V5X3Bhc3N3b3JkLnR4dApNQU5JRkVTVF9QQVRIPSQxCklNQUdFX1JFRkVSRU5DRT0iJFJFRkVSRU5DRSIKRklOR0VSUFJJTlQ9IiRQVUxQX1NJR05JTkdfS0VZX0ZJTkdFUlBSSU5UIgpTSUdOQVRVUkVfUEFUSD0iJFNJR19QQVRIIgojIElNUE9SVEFOVDogU2tvcGVvIGRvZXNudCBhbGxvdyB0byBzZXQgY3VzdG9tIGdudXBnaG9tZSBvciBrZXlyaW5nCiMgU28gaXQgd2lsbCB1c2UgdGhlIGN1cnJlbnQgdXNlciBIT01FLy5nbnVwZyBhbmQgZGVmYXVsdCBrZXlyaW5nCnNrb3BlbyBzdGFuZGFsb25lLXNpZ24gLS1wYXNzcGhyYXNlLWZpbGUgL3RtcC9rZXlfcGFzc3dvcmQudHh0IFwKICAkTUFOSUZFU1RfUEFUSCAkSU1BR0VfUkVGRVJFTkNFICRGSU5HRVJQUklOVCAtLW91dHB1dCAkU0lHTkFUVVJFX1BBVEgKU1RBVFVTPSQ/CmlmIFsgJFNUQVRVUyAtZXEgMCBdOyB0aGVuCiAgZWNobyB7XCJzaWduYXR1cmVfcGF0aFwiOiBcIiRTSUdOQVRVUkVfUEFUSFwifQplbHNlCiAgZXhpdCAkU1RBVFVTCmZp' > /var/lib/pulp/scripts/container_sign.sh;
```

Comment:
decoded
```bash
#!/usr/bin/env bash
echo "Galaxy2024" > /tmp/key_password.txt
MANIFEST_PATH=$1
IMAGE_REFERENCE="$REFERENCE"
FINGERPRINT="$PULP_SIGNING_KEY_FINGERPRINT"
SIGNATURE_PATH="$SIG_PATH"
# IMPORTANT: Skopeo doesnt allow to set custom gnupghome or keyring
# So it will use the current user HOME/.gnupg and default keyring
skopeo standalone-sign --passphrase-file /tmp/key_password.txt \
  $MANIFEST_PATH $IMAGE_REFERENCE $FINGERPRINT --output $SIGNATURE_PATH
STATUS=$?
if [ $STATUS -eq 0 ]; then
  echo {\"signature_path\": \"$SIGNATURE_PATH\"}
else
  exit $STATUS
fi
```